src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -6,6 +6,8 @@
 #include <winrt/TerminalApp.h>
 #include "../../cascadia/inc/cppwinrt_utils.h"
 
+#define CM_NOTIFY_FROM_TRAY (WM_APP + 1)

```

Comment:
This is really our first custom message? I could have sworn we had more somewhere already...

Reply:
I found one in `BaseWindow.h`, I'll put it right under there. Unless there's an even bigger list? 🤔